### PR TITLE
WW-569 menu styling

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

File: extensions/wikia/RWEPageHeader/RWEPageHeaderHooks.class.php [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?php
+
+class RWEPageHeaderHooks {
+	public static function onBeforePageDisplay( \OutputPage $out, \Skin $skin ) {
```
Comment (WikiaTech) [inline extensions/wikia/RWEPageHeader/RWEPageHeaderHooks.class.php]: MAJOR Remove the unused function parameters "$out, $skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	$(function () {
+		$('.rwe-page-header--dropdown').hover(function () {
+			$(this).parent('.rwe-page-header__nav-element')
+				.addClass('rwe-page-header__nav-element--active');
+		}, function () {
+			$(this).parent('.rwe-page-header__nav-element')
+				.removeClass('rwe-page-header__nav-element--active');
+		});
+
+		$('.rwe-page-header__nav-link').on('click', function (e) {
```
Comment (WikiaTech) [inline extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js]: MAJOR Remove the unused function parameter "e". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	};
+
+	$(function () {
+		$('.rwe-page-header--dropdown').hover(function () {
+			$(this).addClass('rwe-page-header--active');
+		}, function () {
+			$(this).removeClass('rwe-page-header--active');
+		});
+
+		$('.rwe-page-header-nav__link').on('click', function (e) {
```
Comment (WikiaTech) [inline extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js]: MAJOR Remove the unused function parameter "e". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Review by rybmat — APPROVED** [review]

**Review by lukaszkonieczny — APPROVED** [review]

**Pull request merged**